client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/index.tsx

```diff
@@ -46,6 +57,41 @@ const SiteContent = ( { data, isLoading, currentPage, isFavoritesTab }: Props, r
 
 	const firstColumn = siteColumns[ 0 ];
 
+	const { data: products } = useProductsQuery();
+
+	const currentLicenseProductSlug = currentLicenseInfo
+		? getProductSlugFromProductType( currentLicenseInfo )
+		: null;
+
+	const currentLicenseProduct = useMemo( () => {
+		return currentLicenseProductSlug && products
+			? products.find( ( product ) => product.slug === currentLicenseProductSlug )
+			: null;
+	}, [ currentLicenseProductSlug, products ] );
+
+	const onIssueLicense = () => {
+		if ( currentLicenseProductSlug ) {
+			recordEvent( 'calypso_jetpack_agency_dashboard_issue_license_info', {
+				products: currentLicenseProductSlug,
+			} );
+			hideLicenseInfo();
+			page(
+				addQueryArgs(
+					{
+						product_slug: currentLicenseProductSlug,
+						source: 'dashboard',
+					},
+					'/partner-portal/issue-license/'
+				)
+			);
+		}
+	};
+
+	const onHideLicenseInfo = () => {
+		recordEvent( 'calypso_jetpack_agency_dashboard_hide_license_info' );
```

Comment:
This hook already appends `calypso_jetpack_agency_dashboard_`. So we can keep it `hide_license_info`

```suggestion
		recordEvent( 'hide_license_info' );
```

Reply:
Do we need this track event? I see this gets tracked even when clicked on "Issue Licenses"

Reply:
You are right. I don't think we need to track this. I have removed it. 👍